packages/itwin/measure-tools/package.json

```diff
@@ -47,41 +47,41 @@
     "!lib/**/test/**/*"
   ],
   "peerDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
```

Comment:
I would make sure to keep the hat `^` before the version number in the `peerDependencies` list. Otherwise that forces consuming applications to use the **exact** same version of those packages which is not something we ever want to do.

For the `devDependencies`, it's okay not to have it.
I would also suggest bumping the `version` of this package in the same PR to ensure there can't be any versioning problems. We're still prerelease (major < 1), but maybe bumping from `0.1.x` to `0.2.x` would be a good indicator that there's been some significant changes in terms of requirements/dependencies and/or code.

Reply:
changed